web/src/components/FOI/Home/Redlining.js

```diff
@@ -780,6 +780,8 @@ const Redlining = React.forwardRef(
                     ];
                   individualPageNo = displayedDoc.page;
                   if (annotations[i]?.type === "fullPage") {
+                    annotations[i].NoResize = true;
+                    annotations[i].NoMove = true;

```

Comment:
is line 784 still neccessary?

Reply:
NoResize is required as its only for fullpage redactions . Removed NoMove .